services/auth/reverseproxy.go

```diff
@@ -105,9 +105,18 @@ func (r *ReverseProxy) newUser(req *http.Request) *user_model.User {
 		}
 	}
 
+	var fullname string
+	if setting.Service.EnableReverseProxyFullName {
+		webAuthFullName := req.Header.Get(setting.ReverseProxyAuthFullName)
```

Comment:
The local variable `webAuthFullName` is unnecessary, just use `fullname` is OK.

Reply:
done